mobify/progressive-web-scaffold
# PLP Refactoring

Refactor some of the elements, selectors and styles of the PLP.

 **Linked PRs**: #151

## Changes
- Refactor PLP and PDP (added to cart modal) to use the ProductItem component
- Allow ProductItem to accept an entire Image component as a prop
- Tweak markup and styles a bit

## Todo
- [x] Merge #151 

## How to test-drive this PR
- `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=http%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Visit a [PLP with items](http://www.merlinspotions.com/potions.html)
- Visit a [PLP with **no** items](http://www.merlinspotions.com/supplies.html)
- Visit a PDP and add it to the cart (observe the "Added to Cart" modal)
- Ensure the appearance matches the mockups
